metricbeat/module/system/process/_meta/data.json

```diff
@@ -11,221 +11,319 @@
+            "-E",
+            "setup.ilm.enabled=false",
+            "-E",
+            "setup.template.enabled=false",
+            "-E",
+            "management.enabled=true",
+            "-E",
+            "logging.level=debug",
+            "-E",
+            "gc_percent=${METRICBEAT_GOGC:100}",
+            "-E",
+            "logging.level=info",
+            "-E",
+            "http.enabled=true",
+            "-E",
+            "http.host=unix:///usr/share/elastic-agent/data/tmp/default/metricbeat/metricbeat.sock_monitor",
+            "-E",
+            "logging.json=true",
+            "-E",
+            "logging.ecs=true",
+            "-E",
+            "logging.files.path=/usr/share/elastic-agent/data/logs/default",
+            "-E",
+            "logging.files.name=metricbeat_monitor-json.log",
+            "-E",
+            "logging.files.keepfiles=7",
+            "-E",
+            "logging.files.permission=0640",
+            "-E",
+            "logging.files.interval=1h",
+            "-E",
+            "path.data=/usr/share/elastic-agent/data/run/default/metricbeat--7.15.0-SNAPSHOT--36643631373035623733363936343635"
         ],
-        "command_line": "/usr/lib/systemd/systemd --switched-root --system --deserialize 29",
+        "command_line": "/usr/share/elastic-agent/data/install/metricbeat-7.15.0-SNAPSHOT-linux-x86_64/metricbeat -E setup.ilm.enabled=false -E setup.template.enabled=false -E management.enabled=true -E logging.level=debug -E gc_percent=${METRICBEAT_GOGC:100} -E logging.level=info -E http.enabled=true -E http.host=unix:///usr/share/elastic-agent/data/tmp/default/metricbeat/metricbeat.sock_monitor -E logging.json=true -E logging.ecs=true -E logging.files.path=/usr/share/elastic-agent/data/logs/default -E logging.files.name=metricbeat_monitor-json.log -E logging.files.keepfiles=7 -E logging.files.permission=0640 -E logging.files.interval=1h -E path.data=/usr/share/elastic-agent/data/run/default/metricbeat--7.15.0-SNAPSHOT--36643631373035623733363936343635",
         "cpu": {
-            "pct": 0.0029,
-            "start_time": "2021-03-29T04:24:52.000Z"
+            "pct": 0.0001,
+            "start_time": "2021-08-05T20:26:34.000Z"
         },
+        "executable": "/usr/share/elastic-agent/data/install/metricbeat-7.15.0-SNAPSHOT-linux-x86_64/metricbeat",
         "memory": {
-            "pct": 0.0011
+            "pct": 0.0023
         },
-        "name": "systemd",
-        "pgid": 1,
-        "pid": 1,
-        "ppid": 0,
-        "state": "sleeping"
+        "name": "metricbeat",
+        "pgid": 201795,
+        "pid": 205480,
+        "ppid": 201795,
+        "state": "sleeping",
+        "working_directory": "/usr/share/elastic-agent/data/install/metricbeat-7.15.0-SNAPSHOT-linux-x86_64"
     },
     "service": {
         "type": "system"
     },
     "system": {
         "process": {
             "cgroup": {
```

Comment:
Is there any field indicating the cgroups version?

Reply:
Ah! Forgot that.